### PR TITLE
Add {{// (note)}} macro

### PR DESCRIPTION
**Pull request by IkariDevGIT**
Wanted to add this new macros to make life easier for card creators

### CODE REVIEW

**Comment by IkariDevGIT** [comment]
With "future updates" i dont mean updates to this PR, this is ready to be merged!

File: public/scripts/templates/macros.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <li><tt>&lcub;&lcub;user&rcub;&rcub;</tt> - your current Persona username</li>
     <li><tt>&lcub;&lcub;char&rcub;&rcub;</tt> - the Character's name</li>
     <li><tt>&lcub;&lcub;input&rcub;&rcub;</tt> - the user input</li>
+    <li><tt>&lcub;&lcub;# (note)&rcub;&rcub;</tt> - you can leave a note here, and the macro will be replaced with blank content</li>
```
Comment (Cohee1207) [inline public/scripts/templates/macros.html]: Doc not updated

Comment (IkariDevGIT) [reply public/scripts/templates/macros.html]: whoops sorry, ill change it

File: public/script.js [hunk 2]
```diff
@@ -1753,11 +1753,14 @@ function substituteParams(content, _name1, _name2, _original, _group) {
     content = content.replace(/<CHARIFNOTGROUP>/gi, _group);
     content = content.replace(/<GROUP>/gi, _group);
 
+    content = content.replace(/\{\{\/\/(.*?)\}\}/g, "");
```
Comment (Cohee1207) [inline public/script.js:1756]: Is this the reverse of the HTML comment? Visible for the user, but not visible for API?

Comment (IkariDevGIT) [reply public/script.js:1756]: yep, now charcard makers can place comments into their card

**Comment by IkariDevGIT** [comment]
so can this be merged?

**Comment by Cohee1207** [comment]
Ok, it can be

**Pull request merged**